src/main/kotlin/no/nav/helsearbeidsgiver/utils/InnsendingUtils.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.helsearbeidsgiver.utils
+
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Avsender
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Inntektsmelding
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Sykmeldt
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.api.AvsenderSystem
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.api.Innsending
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.skjema.SkjemaInntektsmelding
+import no.nav.helsearbeidsgiver.forespoersel.Forespoersel
+import no.nav.helsearbeidsgiver.inntektsmelding.InntektsmeldingRequest
+import no.nav.helsearbeidsgiver.inntektsmelding.InntektsmeldingResponse
+import no.nav.helsearbeidsgiver.utils.wrapper.Fnr
+import no.nav.helsearbeidsgiver.utils.wrapper.Orgnr
+import java.time.OffsetDateTime
+import java.util.UUID
+
+fun SkjemaInntektsmelding.erDuplikat(other: SkjemaInntektsmelding) =
+    this ==
+        other.copy(
+            avsenderTlf = avsenderTlf,
+        )
+
+fun InntektsmeldingRequest.tilInntektsmelding(
+    sluttbrukerOrgnr: Orgnr,
+    lpsOrgnr: Orgnr,
+    forespoersel: Forespoersel,
+): Inntektsmelding =
+    Inntektsmelding(
+        id = UUID.randomUUID(),
+        type =
+            Inntektsmelding.Type.ForespurtEkstern(
+                navReferanseId,
+                AvsenderSystem(
+                    lpsOrgnr,
+                    avsender.systemNavn,
+                    avsender.systemVersjon,
+                ),
+            ),
+        sykmeldt = Sykmeldt(Fnr(sykmeldtFnr), ""),
+        avsender =
+            Avsender(
+                sluttbrukerOrgnr,
+                "",
+                "",
+                arbeidsgiverTlf,
+            ),
+        sykmeldingsperioder = forespoersel.sykmeldingsperioder,
+        agp = agp,
+        inntekt = inntekt,
+        refusjon = refusjon,
+        aarsakInnsending = aarsakInnsending,
+        mottatt = OffsetDateTime.now(),
+        vedtaksperiodeId = null,
```

Comment:
Hvorfor er `vedtaksperiodeId` null?

Reply:
Var null her fordi forespørsel formatet vi bruker i apiet mangler vedtaksperiodeid. Men den er jo i databasen så burde gå an å hente den ut 👍

Reply:
Så lenge det ikke når Simba så burde det være OK for nå.